CPT331.Data.Migration/Migrations/Crime.spGetCrimeOffenceLocalGovernmentAreaState.sql

```diff
@@ -0,0 +1,61 @@
+﻿CREATE PROCEDURE Crime.spGetCrimeOffenceLocalGovernmentAreaState
+
+@Skip INT = 0,
+@Take INT = 100,
+@OrderBy VARCHAR(10) = 'ID',
+@SortDirection VARCHAR(4) = 'ASC'
+
+AS
+
+IF (@OrderBy IS NULL) OR (@OrderBy = '')
+BEGIN
+	SET @OrderBy = 'ID'
+END
+
+IF (@SortDirection IS NULL) OR (@SortDirection = '')
```

Comment:
Based on our current code (```GetCrimeOffenceLocalGovernmentAreaStates()```), I believe it is possible to hit this SP with blanks (_OrderBy only_) or nulls (_OrderBy & SortDirection_).

So it's probably safer to leave this code in there.